index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porfolio project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+ <header class="header-section">
+<a class="logo">My Logo</a>
+<nav> <img class=hamburger src="images/Union.png" alt=""></nav>    
+<!-- <a class="porfolio"href="#">Porfolio</a>
+<a href="#about" class="about">About</a>
+<a href="#contact" class="contact">Contact</a> -->
+</header>
```

Comment:
Good job with the semantic use of header. But kindly remove the commented code and make sure your hamburger nav occurs on the page as the previous reviewer mentioned.
![image](https://user-images.githubusercontent.com/25887253/213511575-f6459541-ca7c-4f11-b800-14c050bbe768.png)